zulip_bots/zulip_bots/lib.py

```diff
@@ -239,9 +240,14 @@ def extract_query_without_mention(message: Dict[str, Any], client: ExternalBotHa
     the stripped message with the bot's @mention removed.  Otherwise, it returns None.
     """
     mention = '@**' + client.full_name + '**'
-    if not message['content'].startswith(mention):
-        return None
-    return message['content'][len(mention):].lstrip()
+    extended_mention_regex = re.compile(r'^@\*\*.*\|' + str(client.user_id) + r'\*\*')
+    print(extended_mention_regex)
```

Comment:
There's a stray `print` here.  Also, I think it's worth saying `content = message['content']` so you don't have to repeat `message['content']` in 4 places.